apis/v1alpha1/ack-generate-metadata.yaml

```diff
@@ -1,13 +1,13 @@
 ack_generate_info:
-  build_date: "2022-09-12T19:22:35Z"
-  build_hash: 2944c8772f216656d84ee02d392eaca501274c1e
-  go_version: go1.17.5
-  version: v0.20.1
-api_directory_checksum: d5cced6600d7ec9d1242ee2f6e96cbb01950bdaa
+  build_date: "2022-11-11T23:46:35Z"
+  build_hash: 18745aa8d1126566776a4748c403ae891b889e9c
+  go_version: go1.18.3
+  version: v0.20.1-7-g18745aa-dirty
```

Comment:
What's going on here?  Don't you want to merge code that's been generated from a release version of code-generator?

Reply:
I'm working on a patch locally (in code-generator) this causes the build script to append the `-dirty` suffix.